verifier/src/main/kotlin/tech/figure/classification/asset/verifier/provenance/AssetClassificationEvent.kt

```diff
@@ -57,17 +57,17 @@ class AssetClassificationEvent(
                             },
                             txHash = sourceTx.txResponse.txhash,
                             eventType = event.type,
-                            attributes = event.attributes.map { attribute ->
+                            attributes = event.attributesList.map { attribute ->
                                 Event(
-                                    key = attribute.key,
-                                    value = attribute.value
+                                    key = attribute.key.toStringUtf8(),
+                                    value = attribute.value.toStringUtf8()
                                 )
                             },
                             fee = sourceTx.tx.authInfo.fee.amountList.firstOrNull()?.amount?.toBigIntegerOrNull(),
                             denom = sourceTx.tx.authInfo.fee.amountList.firstOrNull()?.denom,
                             note = null
                         ),
-                        inputValuesEncoded = false
+                        inputValuesEncoded = true
```

Comment:
The examples I've seen from the txn client have returned the values as encoded for wasm; not sure if rawLogs had them encoded. This will need tested